packages/ecc-client-ga4gh-tes/package.json

```diff
@@ -1,63 +1,57 @@
+  "description": "Web Component for interactive with TES API",
+  "license": "MIT",
+  "version": "0.0.0",
+  "type": "module",
   "main": "./dist/index.js",
   "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
-  "type": "module",
-  "version": "0.0.2",
-  "license": "MIT",
-  "repository": "https://github.com/elixir-cloud-aai/cloud-components",
+  "componentsPrefix": "ecc-client-lit-ga4gh-tes-",
   "exports": {
-    ".": "./dist/index.js",
-    "./package.json": "./package.json"
+    ".": { "types": "./dist/index.d.ts", "import": "./dist/index.js" },
+    "./dist/custom-elements.json": "./dist/custom-elements.json",
+    "./dist/index.js": "./dist/index.js",
+    "./dist/components/*": "./dist/components/*",
+    "./dist/react": "./dist/react/index.js",
+    "./dist/react/*": "./dist/react/*"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elixir-cloud-aai/cloud-components.git"
   },
-  "files": [
-    "images",
-    "dist"
-  ],
-  "keywords": [
-    "TES",
-    "Task Execution Service",
-    "Elixir",
-    "Cloud Component",
-    "Web Component"
-  ],
   "scripts": {
-    "dev": "npm run build && concurrently -k -r \"npm run build -- --watch\" \"wds\"",
-    "lint:fix": "npm run lint -- --fix",
-    "lint": "eslint --ext .ts,.html . --ignore-path .gitignore",
+    "analyze": "cem analyze --litelement",
+    "build": "node ../../scripts/build.js",
+    "dev": "concurrently -r \"npm run build -- --watch\" \"wds\"",
+    "clean": "rm -rf dist node_modules src/react custom-elements-manifest.config.js",
     "test": "wtr --coverage",
     "test:watch": "wtr --watch",
-    "build": "tsup src/index.ts --format cjs,esm --dts",
-    "start:build": "web-dev-server --root-dir dist --app-index index.html --open",
-    "analyze": "cem analyze --fast",
-    "release": "npm run build && npm publish"
-  },
-  "dependencies": {
-    "@microsoft/fast-components": "^2.30.6",
-    "@microsoft/fast-element": "^1.12.0",
-    "@microsoft/fast-foundation": "^2.49.0"
+    "lint": "npx eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "prepublish": "npm run build"
   },
+  "dependencies": { "@elixir-cloud/design": "*", "lit": "^2.8.0" },
   "devDependencies": {
-    "@babel/preset-env": "^7.16.4",
     "@custom-elements-manifest/analyzer": "^0.4.17",
     "@open-wc/eslint-config": "^9.2.1",
     "@open-wc/testing": "^3.1.6",
-    "@typescript-eslint/eslint-plugin": "^5.59.8",
+    "@typescript-eslint/eslint-plugin": "^5.48.0",
+    "@typescript-eslint/parser": "^5.48.0",
     "@web/dev-server": "^0.1.34",
-    "@web/dev-server-esbuild": "^0.4.3",
     "@web/test-runner": "^0.14.0",
-    "babel-plugin-template-html-minifier": "^4.1.0",
     "concurrently": "^5.3.0",
-    "deepmerge": "^4.2.2",
     "eslint": "^8.31.0",
-    "eslint-config-airbnb-typescript": "^17.0.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "rimraf": "^3.0.2",
+    "eslint-config-elixir": "*",
+    "eslint-config-prettier": "^8.3.0",
```

Comment:
Can we have all the common deps as global deps, ie *

Reply:
Yeah, that must be the ideal case! I just don't think it is the best idea to do this in the same PR as this, as that may require more discussion & maybe some changes to the source.